### PR TITLE
don't call typecheck with --tags=dockerless now that dockershim is ac…

### PR DESCRIPTION
**Pull request by BenTheElder**
…tually removed

#### What type of PR is this?

/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

see: https://github.com/kubernetes/kubernetes/pull/97252

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

/sig node
/assign @dims

### CODE REVIEW

File: hack/verify-typecheck-dockerless.sh [hunk 1]
```diff
@@ -19,11 +19,7 @@ set -o nounset
...
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
-
 cd "${KUBE_ROOT}"
-# verify the dockerless build
-# https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/1547-building-kubelet-without-docker/README.md
-hack/verify-typecheck.sh --skip-test --tags=dockerless --ignore-dirs=test
 
 # verify using go list
 if _out="$(go list -mod=readonly -tags "dockerless" -e -json  k8s.io/kubernetes/cmd/kubelet/... \
```
Comment (BenTheElder) [inline hack/verify-typecheck-dockerless.sh:25]: we might? still want to do this, but rename the script?

for now I just dropped the unnecessary additional typecheck pass.

Comment (dims) [reply hack/verify-typecheck-dockerless.sh:25]: let's just drop the entire file?

Comment (pacoxu) [reply hack/verify-typecheck-dockerless.sh:25]: LGTM

Comment (pacoxu) [reply hack/verify-typecheck-dockerless.sh:25]: But it seems that #107119 has done further to remove the entire file with a cleanup in kubelet.

Comment (hakman) [reply hack/verify-typecheck-dockerless.sh:25]: Also, removed from ever running in https://github.com/kubernetes/test-infra/pull/24710.

**Comment by dims** [comment]
let's land https://github.com/kubernetes/kubernetes/pull/107119 instead of this.

thanks @BenTheElder 

/close

**Pull request closed without merging**